test/conformance/kuttl/generate/clusterpolicy/cornercases/pod-restart-on-cm-update/manifests.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-state-metrics
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kube-state-metrics-source-cm
+  namespace: kube-state-metrics
+  labels:
+    kubestatemetrics.platform.example: source
+data:
+  allowed: '"true"'
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: kube-state-metrics
+  name: kube-state-metrics
+  labels:
+    app: nginx
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+        - name: nginx
+          image: nginx:1.14.2
```

Comment:
Throughout tests like these where we are working with Pods, I like to pull a very small and lightweight image. This has the benefit of reducing test times because we aren't waiting an additional 5-10 seconds. I recommend using `busybox` rather than `nginx`. It's also that if we standardize on that, the image will only be pulled once and cached on the KinD nodes in the test cluster, making tests even faster.

Reply:
Yes, I had a doubt on which one to use. Changing to `busybox`.